src/alertsmanagement/azext_alertsmanagement/custom.py

```diff
@@ -0,0 +1,324 @@
+                                       groupby,
+                                       include_smart_groups_count=None,
+                                       target_resource=None,
+                                       target_resource_type=None,
+                                       target_resource_group=None,
+                                       monitor_service=None,
+                                       monitor_condition=None,
+                                       severity=None,
+                                       alert_state=None,
+                                       alert_rule=None,
+                                       time_range=None,
+                                       custom_time_range=None):
+    return client.get_summary(groupby=groupby, include_smart_groups_count=include_smart_groups_count, target_resource=target_resource, target_resource_type=target_resource_type, target_resource_group=target_resource_group, monitor_service=monitor_service, monitor_condition=monitor_condition, severity=severity, alert_state=alert_state, alert_rule=alert_rule, time_range=time_range, custom_time_range=custom_time_range)
+
+
+def change_state_alertsmanagement_smart_group(cmd, client,
+                                              smart_group_id,
+                                              new_state=None):
+    return client.change_state(smart_group_id=smart_group_id, new_state=new_state)
+
+
+def get_all_alertsmanagement_smart_group(cmd, client,
+                                         target_resource=None,
+                                         target_resource_group=None,
+                                         target_resource_type=None,
+                                         monitor_service=None,
+                                         monitor_condition=None,
+                                         severity=None,
+                                         smart_group_state=None,
+                                         time_range=None,
+                                         page_count=None,
+                                         sort_by=None,
+                                         sort_order=None):
+    return client.get_all(target_resource=target_resource, target_resource_group=target_resource_group, target_resource_type=target_resource_type, monitor_service=monitor_service, monitor_condition=monitor_condition, severity=severity, smart_group_state=smart_group_state, time_range=time_range, page_count=page_count, sort_by=sort_by, sort_order=sort_order)
+
+
+def get_by_id_alertsmanagement_smart_group(cmd, client,
+                                           smart_group_id):
+    return client.get_by_id(smart_group_id=smart_group_id)
+
+
+def get_history_alertsmanagement_smart_group(cmd, client,
+                                             smart_group_id):
+    return client.get_history(smart_group_id=smart_group_id)
+
+
+def _transform_condition(condition):
+    """
+    ['Equals', 'Sev0', 'Sev2'] ->
```

Comment:
is there list of supported operator? If yes, let's add it in help message also

Reply:
Yes, there is. It's already in help.